### ISSUE TITLE
FCC logo not linked to curriculum has aria-label "freeCodeCamp Curriculum"

### ISSUE THREAD

**Issue by sidemt** [freeCodeCamp/freeCodeCamp#45095]
The translation key and string for `fcc-logo: freeCodeCamp Logo` were changed to `fcc-curriculum: freeCodeCamp Curriculum` in #45039
As far as I understand, this change was intended for the logo in the navigation bar, but the `FreeCodeCampLogo` component is also used on certificates.

The logo on the certificate is not linked to the curriculum. So aria-label "freeCodeCamp Curriculum" seems incorrect here.

## Affected page

Users' certifications such as: https://www.freecodecamp.org/certification/{username}/responsive-web-design
**This change is not deployed to the production yet. I checked it on Gitpod.**

## To reproduce

1. Go to a certification page
2. Check the aria-label of the freeCodeCamp logo on the certificate

## Expected behavior

aria-label should be "freeCodeCamp Logo" when it is not used as a link
(I'm not sure if this is the best solution)

## Screenshots

**This change is not deployed to the production yet. I checked it on Gitpod.**
![fcc-logo](https://user-images.githubusercontent.com/25644062/153718473-f8c0ecd1-f5ce-4004-963d-4336cb01eef8.png)

## System

- Device: Laptop
- OS: Windows 10
- Browser: Brave
- Version: v 1.35.101
 
## Additional context

Originally pointed out in a Crowdin comment:
https://freecodecamp.crowdin.com/translate/249b43ffcacdb664884a87464db3db6d/25826/en-ja#1142854

**Comment by huyenltnguyen** [comment]
To fix the issue, I think we should:
- Add back the `fcc-logo` string
- Allow the `FreeCodeCampLogo` component to accept an `ariaLabel` prop
- Pass the appropriate string as `ariaLabel` to `FreeCodeCampLogo` from the components that use it
